packages/router/README.md

```diff
@@ -143,6 +128,41 @@ becomes...
 </MainLayout>
 ```
 
+### `private` Set
+
+All `Routes` nested in `<Private>` require authentication.
+
+Sets can take a `private` prop which makes all routes inside that set require authentication.  When a user is not authenticated and attempts to visit this route, they will be redirected to the route passed as the `unauthenticated` prop and the originally requested route's path will be added to the query string in a `redirectTo` param. This lets you send the user to the originally requested page once logged in. Additionally you can also specify `role` to be more fine-grained in your access control.
+
+This is an example of how you'd use a private set:
+
+```jsx
+// Routes.js
+<Router>
+  <Route path="/" page={HomePage} name="home" />
+  <Set private unauthenticated="home">
+    <Route path="/admin" page={AdminPage} name="admin" />
+  </Set>
+</Router>
+```
+
+Private routes are important, and should be easy to spot in your routes file. The larger your routes file gets, the more difficult it will probably become to find `<Set private /*...*/>` among your other sets. So we also provide a `<Private>` component that's just an alias for `<Set private /*...*/>`. Most of our documentation will use `<Private>`.
```

Comment:
- rm comma (no particular reason, just already a short sentence; the thoughts are related)
- present tense here
```suggestion
Private routes are important and should be easy to spot in your Routes file. The larger your Routes file gets, the more difficult it will probably become to find `<Set private /*...*/>` among your other Sets. So we also provide a `<Private>` component that's just an alias for `<Set private /*...*/>`. Most of our documentation uses `<Private>`.
```